packages/css-modules/index.js

```diff
@@ -23,18 +23,14 @@ function cssModules (options) {
   const importLoaders = options.importLoaders || 1
   const localIdentName = options.localIdentName || localIdentDefault
 
-  return (context) => ({
-    module: {
-      loaders: [
-        Object.assign({
-          test: context.fileType('text/css'),
-          loaders: [ 'style-loader', 'css-loader?' + stringifyQueryParams({ importLoaders, localIdentName, modules: true }) ]
-        }, options.exclude ? {
-          exclude: Array.isArray(options.exclude) ? options.exclude : [ options.exclude ]
-        } : {})
-      ]
-    }
-  })
+  return (context, util) => util.addLoader(
+    Object.assign({
+      test: context.fileType('text/css'),
```

Comment:
If not, use a post hook to query all loaders with style loader and use that mime type